tests/integration/test_charm.py

```diff
@@ -177,3 +261,27 @@ async def pull_content_from_unit_file(unit, path: str) -> str:
     """
     action = await unit.run(f"cat {path}")
     return action.results.get("Stdout", None)
+
+
+async def wait_for_status(ops_test, status: str, wait: float):
```

Comment:
is `wait_for_status` being used anywhere? if not, should we delete it?

Reply:
Nope, this is totally unrelated and shouldn't be here.